pinot-controller/src/main/resources/app/pages/InstanceDetails.tsx

```diff
@@ -308,7 +315,7 @@ const InstanceDetails = ({ match }: RouteComponentProps<Props>) => {
               </CustomButton>
               <CustomButton
                 onClick={handleDropAction}
-                tooltipTitle="Removes the node from the cluster. Untag & rebalance (to ensure node is not being used by any table), and shutdown instance, before dropping."
+                tooltipTitle={!instanceName.startsWith('Minion_') ? "Removes the node from the cluster. Untag & rebalance (to ensure node is not being used by any table), and shutdown instance, before dropping." : ""}
```

Comment:
Removes the node from the cluster. Untag and rebalance (to ensure the node is not being used by any table) and shutdown the instance before dropping.

@npawar

Reply:
Is there a difference between line 74

```
else if (instanceName.toLowerCase().startsWith('minion')) {
    instanceType = 'MINION';
  }
```

and the condition here? Could we just use `instanceType === 'MINION'` here?